sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -155,14 +172,28 @@ public void close() {
     /**
      * Completes a {@link ServiceBusMessage} using its lock token. This will delete the message from the service.
      *
-     * @param message to be used.
+     * @param message Message to be completed.
      *
      * @return The {@link Mono} the finishes this operation on service bus resource.
      */
     public Mono<Void> complete(ServiceBusReceivedMessage message) {
```

Comment:
Do you have test when user do not want autoComplete and  want to call him self ?

Reply:
I'll add that.